apps/extension/src/shared/components/default-frontend-form/new-frontend-input.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  FocusEventHandler,
+  ForwardedRef,
+  forwardRef,
+  KeyboardEventHandler,
+  MutableRefObject,
+  useRef,
+  useState,
+} from 'react';
+import { SelectList } from '@penumbra-zone/ui/components/ui/select-list';
+import { cn } from '@penumbra-zone/ui/lib/utils';
+import { isValidUrl } from '../../utils/is-valid-url';
+
+const getUrlValidity = (url: string, initialUrl?: string): boolean => {
+  return isValidUrl(url) && url !== initialUrl;
+};
```

Comment:
Can we rename `getUrlValidity` to something like `isValidAndNotInitial()`? So it makes the boolean return make more sense?